admin/controllers/pupil-pin.js

```diff
@@ -24,7 +24,7 @@ const getGeneratePinsOverview = async (req, res, next) => {
   let pupils
   try {
     if (featureToggles.isFeatureEnabled('prepareCheckMessaging')) {
-      pupils = await pinGenerationV2Service.getPupilsWithActivePins(req.user.schoolId, pinEnv)
+      pupils = await pinGenerationV2Service.getPupilsWithActivePins(req.user.schoolId, pinEnv === 'live')
```

Comment:
why are we hardcoding this to live?

Reply:
It's an expression that returns true or false, as the parameter is a boolean.

😀 I assume this is a pre-coffee review 😀

Reply:
brewing now 🤦‍♂️